apps/client/src/components/date/date.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from 'react';
+import styles from './date.module.css';
+
+const GetDate: React.FC = () => {
+  const [date, setDate] = useState(new Date());
+
+  useEffect(() => {
+    const timer = setInterval(() => setDate(new Date()), 1000);
```

Comment:
Render every second is overkill for your project, you should set it to every hour or so⏰